daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -4,10 +4,7 @@
 package com.daml.lf.speedy
 package svalue
 
-import java.util
```

Comment:
Why can we delete all the stuff in here?

Reply:
This is tested extensively in [ComparisonSBuiltinTest](https://github.com/digital-asset/daml/blob/master/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ComparisonSBuiltinTest.scala)